mubench.pipeline/data/detector.py

```diff
@@ -2,21 +2,20 @@
 from typing import Optional, List
 
 from data.finding import Finding, SpecializedFinding
-
+from data.java_interface import JavaInterface
 
 class Detector:
     BASE_URL = "http://www.st.informatik.tu-darmstadt.de/artifacts/mubench/detectors"
-    CLI_VERSION = "v20170406"
 
-    def __init__(self, detectors_path: str, detector_id: str, java_options: List[str]):
+    def __init__(self, detectors_path: str, detector_id: str, java_options: List[str], cli_version: str = "20170406"):
```

Comment:
The version was "**v**20170406" before. Please fix this. We should switch this to use the version of the CLI maven artifact, we can do this in our meeting tomorrow.